spec.html

```diff
@@ -36709,7 +36168,6 @@ <h1>Properties of the Set Constructor</h1>
 
       <emu-clause id="sec-set.prototype">
         <h1>Set.prototype</h1>
-        <p>The initial value of `Set.prototype` is the intrinsic %SetPrototype% object.</p>
```

Comment:
You eliminated this line, and the corresponding one for `WeakSet.prototype`, which makes sense because they're tautologies now. But why just these two? There are about 20 others.

Reply:
#2131 should address these; i'll rebase this after that lands and clean it up.